entity-framework/core/extensions/index.md

```diff
@@ -41,7 +41,7 @@ A Visual Studio 2017 extension that adds an ORM designer for visual design of En
 
 ### CatFactory
 
-CatFactory is a scaffolding engine for .NET Core and Entity Framework Core. The concept behind CatFactory it's export an existing database from SQL Server instance, then with the representation in models for database; scaffold entities, configurations, repositories and more.
+CatFactory is a scaffolding engine for .NET Core and Entity Framework Core. The concept behind CatFactory is to export an existing database from a SQL Server instance, then with the representation in models for database; scaffold entities, configurations, repositories and more.
```

Comment:
Thanks @ErikEJ for keeping this in the radar! Unfortunately it is still very vague, the use of semicolon doesn't make any sense to me; but I don't know how to fix it :smile:
 
I looked at the explanation in the readme at https://github.com/hherzl/CatFactory.EntityFrameworkCore, and although I am not familiar with CatFactory, it seems much better to me. 

Given we already include a link, I suggest we simplify our description to something like this:

> CatFactory is a scaffolding engine for .NET Core that can automate the generation of EF Core-compatible entities, entity configurations, DbContext and repositories types from a SQL Server database.